pytorch_blade/tests/disc/ops/test_activation.py

```diff
@@ -55,12 +55,13 @@ def test_gelu_static_shape(self):
     def test_gelu_dynamic_shape(self):
         self._test_activation(torch.nn.GELU(), torch.nn.functional.gelu, [([-1, -1, -1, -1], torch.float)])
 
-
-    @skipIfEnableTorchMlir()
-    #TODO(yancey1989): dependence binary op
-    def test_hardtanh(self):
+    def test_hardtanh_static_shape(self):
```

Comment:
Add empty format test?

Reply:
![image](https://user-images.githubusercontent.com/8392447/175449354-2917b4a2-6eaa-44a8-aea5-5d64184b57c7.png)

Reply:
I removed the empty tensor lowering, the empty tensor is lowered from the official decompose pass, this action is not as disc expected, so I added a `DiscDecompose` pass in front of the official decompose pass.